NTO/Documents/attributes/documentNumber.ttl

```diff
@@ -0,0 +1,14 @@
+@prefix ogit.Documents:        <http://www.purl.org/ogit/Documents/> .
+@prefix ogit:                   <http://www.purl.org/ogit/> .
+@prefix owl:                    <http://www.w3.org/2002/07/owl#> .
+@prefix rdfs:                   <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix dcterms:                <http://purl.org/dc/terms/> .
+
+ogit.Documents:documentNumber
+	a owl:DatatypeProperty;
+	rdfs:subPropertyOf ogit:Attribute;
+	rdfs:label "documentNumber";
+	dcterms:description "Contains the number of a document.";
+	dcterms:valid "start=2020-04-16;";
```

Comment:
its 2021 now :P

It's the same in some other files as well.

Reply:
Fixed for all four attributes.